test/test.js

```diff
@@ -421,15 +427,48 @@ ruleTester.run('generated-flow-types', rules['generated-flow-types'], {
   invalid: [
     {
       code: `
-        import type {TestFragment_other} from './path/to/TestFragment_other.graphql';
+        import type {TestFragment_other$ref} from './path/to/TestFragment_other.graphql';
```

Comment:
at first it wasn't clear why this was erroring, maybe make it something like TestFragment_OTHER$ref to make it more obvious?